src/security/pages/sign-up.component.vue

```diff
@@ -0,0 +1,59 @@
+<script setup>
+import InputText from "primevue/inputtext";
+import ToggleButton from "primevue/togglebutton";
+import { ref } from "vue";
+
+const checked1 = ref(false);
+</script>
+
+<template>
+  <div class="flex justify-center items-center">
+    <div class="flex flex-col container max-w-md items-center mt-8">
+      <div class="py-6">
+        <span class="text-2xl font-semibold">Create an account</span>
+      </div>
+      <span class="p-float-label w-full">
+        <InputText id="firstname" type="text" class="rounded-xl my-4 w-full" />
+        <label for="firstname">First name</label>
+      </span>
+      <span class="p-float-label w-full">
+        <InputText id="lastname" type="text" class="rounded-xl my-4 w-full" />
+        <label for="lastname">Last name</label>
+      </span>
+      <span class="p-float-label w-full">
+        <InputText id="email" type="email" class="rounded-xl my-4 w-full" />
+        <label for="email">Email</label>
+      </span>
+      <span class="p-float-label w-full">
+        <InputText
+          id="password"
+          type="password"
+          class="rounded-xl my-4 w-full" />
+        <label for="password">Password</label>
+      </span>
+      <span class="p-float-label w-full">
+        <InputText id="phone" type="tel" class="rounded-xl my-4 w-full" />
```

Comment:
This one would work better with an [InputMask](https://primefaces.org/primevue/inputmask).

Reply:
I'll keep using InputText because it seems like InputMask does not support Peruvian phone numbers, hahaha.

Reply:
Nevermind, it does